modules/highgui/src/window.cpp

```diff
@@ -481,12 +486,12 @@ int cv::createButton(const String&, ButtonCallback, void*, int , bool )
 
 #endif
 
-#if   defined (HAVE_WIN32UI)  // see window_w32.cpp
-#elif defined (HAVE_GTK)      // see window_gtk.cpp
-#elif defined (HAVE_COCOA)    // see window_carbon.cpp
+#if   defined (HAVE_WIN32UI)                    // see window_w32.cpp
+#elif defined (HAVE_GTK)                        // see window_gtk.cpp
+#elif defined (HAVE_COCOA)                      // see window_carbon.cpp
 #elif defined (HAVE_CARBON)
-#elif defined (HAVE_QT)       // see window_QT.cpp
-#elif defined (WINRT) && !defined (WINRT_8_0) // see window_winrt.cpp
+#elif defined (HAVE_QT)                         // see window_QT.cpp
+#elif defined (WINRT) && !defined (WINRT_8_0)   // see window_winrt.cpp
```

Comment:
There are no reason to touch so many lines. What is changed here except whitespaces?